packages/auth/src/grant/routes.ts

```diff
@@ -42,7 +48,8 @@ export function createGrantRoutes({
     interaction: {
       start: (ctx: AppContext) => startInteraction(deps, ctx),
       finish: (ctx: AppContext) => finishInteraction(deps, ctx)
-    }
+    },
+    post: (ctx: AppContext) => continueGrant(deps, ctx)
```

Comment:
Can we call that `continue`? That way, it can easily be included in the mapping functionality in app.ts (the one from spec to code I commented on in app.ts).